guides/v3.4.0/models/index.md

```diff
@@ -1,5 +1,19 @@
-Models are objects that represent the underlying data that your
-application presents to the user. Different apps will have very
+Ember developers have great options for how they handle data from back end APIs. 
+Ember itself works with any type of back end: REST, JSON API, GraphQL, or anything else.
+
+Many developers choose to use Ember Data, a powerful set of tools for formatting requests, serializing responses, and efficiently managing a local cache of data. 
+The Ember Data library is included by default in Ember apps. 
+Alternately, some developers write all their own code to handle API requests, using native JavaScript methods like [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) or third-party libraries.
+Many apps use a combination of approaches.
+
+This section of the Guides describes the essential features of Ember Data. To learn about other ways to handle data in an app, check out [Ember Observer](https://www.emberobserver.com/) for addons or search for community-made tutorials.
+
+## What are Ember Data models?
+
+In Ember Data, models are objects that represent the underlying data that your
+application presents to the user. 
+(Note that Ember Data models are a different concept than the `model` hook method on Routes, although they share the same name.)
```

Comment:
Although generally speaking, isn't the purpose of the `model` hook to load the ember-data `model`(s) for presentation?  The note seems to suggest they are not related, even though it is fair to say they are linked ("Typically, the model hook should return an Ember Data record" from route model guide).  Also, in the route model guide, when it references the ember-data record, it links to this page, so maybe this parenthetical is unnecessary.

Reply:
Yet that is what the guide says - "Typically, the model hook should return an Ember Data record"

Reply:
“Typically, the model hook should return an Ember Data record, but it can also return any promise object (Ember Data records are promises), or a plain JavaScript object or array.“ [link](
https://guides.emberjs.com/release/routing/specifying-a-routes-model/)

Is there a way the quote above from the models hook guide can be improved?

Given the amount of confusion that is common with new devs, I’m ok with repeating the point. I’ll read over the model hook docs and open issues for possible improvements.

However I should probably link to the model hook info here.

Reply:
I have removed the parens, added a link, and used the word “different” concept instead of “separate”